oci/oci.go

```diff
@@ -106,12 +106,6 @@ func (r *Runtime) Name() string {
 // this will return either the trusted or untrusted runtime path.
 func (r *Runtime) Path(c *Container) string {
 	if !c.trusted {
-		// We have an explicitly untrusted container.
-		if c.privileged {
-			logrus.Warnf("Running an untrusted but privileged container")
-			return r.trustedPath
-		}
-
```

Comment:
Just to clarify, a bit below (line 124) you're not changing the condition
```go
if c.privileged || r.untrustedPath == "" {
```
into 
```
if r.untrustedPath == "" {
```
because you don't want the untrusted runtime to be picked if the container was not explicitly tagged as `untrusted` AND `is_privileged`, right ?

Reply:
correct.  if they didn't explicitly set the annotation, we should use the default runtime.  I updated the commit message to clarify this - PTAL and let me know if that isn't clear now.

Reply:
Yes this is fine, thanks for clarifying!